tests/falco/commands_test.go

```diff
@@ -53,15 +53,27 @@ func TestFalco_Cmd_Version(t *testing.T) {
 		res := falco.Test(runner, falco.WithArgs("--version"))
 		assert.NoError(t, res.Err(), "%s", res.Stderr())
 		assert.Equal(t, res.ExitCode(), 0)
+		// Falco version supports:
+		// - (dev) -> 0.36.0-198+30aa28f
+		// - (release) -> 0.36.0
+		// - (release-rc) -> 0.36.0-rc1
+		// Libs version supports:
+		// - (commit hash) -> e999e61fa8f57ca8e9590e4c108fd4a12459ec48
+		// - (release) -> 0.13.0
+		// - (release-rc) -> 0.13.0-rc1
+		// Default driver supports:
+		// - (commit hash) -> e999e61fa8f57ca8e9590e4c108fd4a12459ec48
+		// - (release) -> 6.0.1+driver
+		// - (release-rc) -> 6.0.1+driver-rc1
 		assert.Regexp(t, regexp.MustCompile(
-			`Falco version:[\s]+(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?[\s]+`+
```

Comment:
This is actually the standard semver regex that you can find on https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string, I don't think we need to change it. However, the addition for the hash below is definitely important :)

Reply:
Uhm you are right! In the end, the Falco version cannot be a pure hash commit, i will change it, thank you!